python/vyos/frrender.py

```diff
@@ -147,6 +148,43 @@ def dict_helper_pim_defaults(pim, path):
         pim = config_dict_merge(default_values, pim)
         return pim
 
+    def dict_helper_nhrp_defaults(nhrp):
+        nflog_redirect = 1
```

Comment:
What are those magic numbers? Please add a proper comment.

Reply:
I have added comments about these two numbers.
https://github.com/vyos/vyos-1x/pull/4217/files#diff-5fa707da9950d5de7cf947091ea150f12fd562a821f70857f1733ef1bcab9091R152-R155

More about NFLOG in FRR is described in FRR documentation